lms/static/js/groups/models/course_cohort_settings.js

```diff
@@ -7,7 +7,7 @@
                 is_cohorted: false,
                 cohorted_inline_discussions: [],
                 cohorted_course_wide_discussions: [],
-                always_cohort_inline_discussions: true
+                always_cohort_inline_discussions: false
```

Comment:
Note to reviewers: the default value is sprinkled throughout (to try to represent what is expected, I guess). In reality these JS default values (here and below) are not used because the data is always passed from the server, but changing the default for consistency.